.github/workflows/main.yml

```diff
@@ -6,7 +6,23 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the develop branch
   push:
+    branches:
+      - main
+      - dev # branch used for testing incremental builds
+    paths-ignore:
+      - 'devResources/**'
+      - 'docker-compose.yml'
+      - '.gitignore'
+      - 'LICENSE.md'
+      - 'README.md'
+  pull_request:
     branches: [ main ]
+    paths-ignore:
+      - 'devResources/**'
+      - 'docker-compose.yml'
```

Comment:
Changes to `devResources` and `docker-compose.yml` shouldn't be ignored--any changes here should trigger a new build

Reply:
resolved as per https://github.com/geoCML/geocml-base-deployment/pull/61/commits/dbe275dc9b60dee44789c1c8c1d77b76432f376b